x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/types.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface Fields {
+  filterFields: string[];
+  sortFields: string[];
+}
+
+export type SortDirection = 'asc' | 'desc';
```

Comment:
[blocking request] So I noticed this when looking at the tests, but unfortunately I do think we need to rework this to be i18n'able instead of flat strings. Non-English users will probably not understand what `asc` or `desc` mean.

We might want to use an enum here maybe? Not sure, what do you think?

```ts
export enum SortDirection {
  ASC = i18n.translate(
    'xpack.enterpriseSearch.appSearch.documents.search.sortDirection.ascendingDropDownOptionLabel',
    { defaultMessage: 'asc' }
  ),
  DESC = i18n.translate(
    'xpack.enterpriseSearch.appSearch.documents.search.sortDirection.descendingDropDownOptionLabel',
    { defaultMessage: 'desc' }
  ),
}
```

Reply:
https://github.com/elastic/kibana/pull/88491/commits/ef838fda86e84492c148d36391152f190896ef06